userspace/libscap/CMakeLists.txt

```diff
@@ -214,21 +214,7 @@ endif()
 
 if(HAS_ENGINE_GVISOR)
 	add_subdirectory(engine/gvisor)
-	# The static and shared build differs here because a shared scap_engine_gvisor
-	# will result in circular dependencies.
-	if(BUILD_SHARED_LIBS)
```

Comment:
Just a question after a very first look: will `BUILD_SHARED_LIBS` behave like before? :thinking:

Reply:
WDYM? libscap_engine_gvisor.so becomes a separate shared library, so that's a change, but other than that, there should be no observable changes (at least, Works For Me :tm:)

TBH I'm not entirely sure when the circular dependency disappeared, probably when extracting some scap_platform_util or other

Reply:
I meant regarding `BUILD_SHARED_LIBS` in general, but I noticed right now that it's only related to gvisor (and I believe it's acceptable).

cc @LucaGuerra

Reply:
FWIW, I consider splitting out scap_engine_gvisor as a separate .so a fix

Reply:
sounds fine!